src/Navbar.tsx

```diff
@@ -1,12 +1,38 @@
+import { NavLink } from "react-router-dom";
```

Comment:
Shriyan worked on the Navbar in a separate branch, so I would probably just revert this so you don't have conflicting changes.